WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationSettingsViewController.swift

```diff
@@ -55,9 +55,12 @@ open class NotificationSettingsViewController: UIViewController {
         // Style!
         WPStyleGuide.configureColors(view: view, tableView: tableView)
         WPStyleGuide.configureAutomaticHeightRows(for: tableView)
+
+        activityIndicatorView.tintColor = .textSubtle
     }
 
 
+
```

Comment:
Just wanted to point out the extra empty line may not be necessary.

Apologies for reviewing the whitespace though.  Treat this as optional.